cmd/ci-operator-configresolver/main.go

```diff
@@ -128,6 +130,13 @@ func validateOptions(o options) error {
 			return fmt.Errorf("--registry points to a nonexistent directory: %v", err)
 		}
 	}
+	if o.prowPath == "" {
```

Comment:
I don't think we can do this, because there's no way to incrementally deploy this without blowing up the tool. Adding a nonexistent flag before we bump will break, too. Choose an incremental process.

Reply:
I've added a fix for this. It breaks the job table (which is just empty without the flag), but everything else still works.